src/examples/candid-health/send-to-candid.test.ts

```diff
@@ -0,0 +1,503 @@
+import {
+  createReference,
+  getReferenceString,
+  indexSearchParameterBundle,
+  indexStructureDefinitionBundle,
+  MedplumClient,
+} from '@medplum/core';
+import { readJson } from '@medplum/definitions';
+import { Bundle, Coverage, Encounter, Patient, SearchParameter } from '@medplum/fhirtypes';
+import { DrAliceSmith, MockClient } from '@medplum/mock';
+import fetch from 'node-fetch';
+import { handler } from './send-to-candid';
+
+vi.mock('node-fetch', () => ({
+  default: vi.fn(() => ({
+    json: () => ({}),
+  })),
+}));
+
+describe('Candid Health Tests', () => {
+  beforeAll(() => {
+    indexStructureDefinitionBundle(readJson('fhir/r4/profiles-types.json') as Bundle);
+    indexStructureDefinitionBundle(readJson('fhir/r4/profiles-resources.json') as Bundle);
+    indexSearchParameterBundle(readJson('fhir/r4/search-parameters.json') as Bundle<SearchParameter>);
+  });
+
+  beforeEach(async (ctx) => {
+    const medplum = new MockClient();
+
+    const patient = await medplum.createResource({
+      resourceType: 'Patient',
+      name: [
+        {
+          family: 'Kirby901',
+          given: ['John43'],
+          prefix: ['Mr.'],
+        },
+      ],
+
+      gender: 'male',
+      birthDate: '1940-09-05',
+      address: [
+        {
+          line: ['599 Schowalter Promenade'],
+          city: 'West Springfield',
+          state: 'MA',
+          postalCode: '01089',
+          country: 'US',
+          period: {
+            start: '1940-09-05',
+          },
+        },
+      ],
+      maritalStatus: {
+        coding: [
+          {
+            system: 'http://terminology.hl7.org/CodeSystem/v3-MaritalStatus',
+            code: 'M',
+            display: 'M',
+          },
+        ],
+        text: 'M',
+      },
+    });
+
+    const serviceProvider = await medplum.createResource({
+      name: 'HOLYOKE MEDICAL CENTER',
+      address: [
+        {
+          line: ['575 BEECH STREET'],
+          city: 'HOLYOKE',
+          state: 'MA',
+          postalCode: '01040-2223',
+          country: 'US',
+        },
+      ],
+      resourceType: 'Organization',
+    });
+
+    const encounter = await medplum.createResource({
```

Comment:
Is this from some kind of IG?  Probably good to link to it if so?

Reply:
Acutally no - this was hand-crafted